sdk/storage/azure-storage-blob/src/test/java/com/azure/storage/blob/AppendBlobAPITest.groovy

```diff
@@ -264,10 +263,10 @@ class AppendBlobAPITest extends APISpec {
 
         when:
         bc.appendBlockWithResponse(defaultInputStream.get(), defaultDataSize, bac, null, null)
-
```

Comment:
Nit: Don't delete these blank lines. Having a separation between labeled blocks is really useful for readability.

Reply:
Will revert the changes

Reply:
Done.